scripts/js-check.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
Since we're using pipes, we probably want `set -euo pipefail`

Reply:
thanks